src/main/java/com/ericsson/esj/JenkinsManager.java

```
@@ -0,0 +1,259 @@
+        HttpRequest httpRequest = new HttpRequest(HttpMethod.POST);
+        boolean success = false;
+
+        if (jobName == null) {
+            jobName = DEFAULT_JOB_NAME;
+        }
+
+        httpRequest.setJenkinsBaseUrl(jenkinsBaseUrl).addHeader("Authorization", "Basic " + encoding)
+                .addHeader("Content-type", MediaType.APPLICATION_XML).addHeader("Jenkins-Crumb", crumb)
+                .addParam("name", jobName).setBody(jobXmlData).setEndpoint("/createItem");
+
+        ResponseEntity response = httpRequest.performRequest();
+        success = response.getStatusCode() == HttpStatus.SC_OK;
+
+        if(!success) {
+            LOGGER.error("Failed to create a jenkins job. Status code: " + response.getStatusCodeValue() + ". Reason: " + response.getBody());
+        }
+
+        return success;
+    }
+
+    /**
+     * This function recieves a jenkins job name and are used to trigger the job
+     *
+     * @throws URISyntaxException
+     */
+    public boolean triggerJob(String jobName, String job_token) throws URISyntaxException {
+        HttpRequest httpRequest = new HttpRequest(HttpMethod.GET);
+        boolean success = false;
+
+        if (jobName == null) {
+            jobName = DEFAULT_JOB_NAME;
+        }
+        if (job_token == null) {
+            job_token = DEFAULT_JOB_TOKEN;
+        }
+
+        String endpoint = "/job/" + jobName + "/build";
+
+        httpRequest.setJenkinsBaseUrl(jenkinsBaseUrl).addHeader("Authorization", "Basic " + encoding)
+                .addHeader("Content-type", MediaType.APPLICATION_JSON).addParam("token", job_token)
+                .setEndpoint(endpoint);
+
+        ResponseEntity response = httpRequest.performRequest();
+        success = response.getStatusCode() == HttpStatus.SC_CREATED;
+
+        if(!success) {
+            LOGGER.error("Failed to trigger a jenkins job. Status code: " + response.getStatusCodeValue() + ". Reason: " + response.getBody());
```

Maybe this should also throw an error if it did not succeed? Then the user may or may not handle it depending on their implementation